pallet/crowdsale/src/types.rs

```diff
@@ -25,6 +25,8 @@ pub struct SaleInformation<AccountId, BlockNumber> {
 	pub soft_cap_price: Balance,
 	/// Total funds raised during the crowdsale
 	pub funds_raised: Balance,
+	/// Total number of unique contributors that have participated in the crowdsale
+	pub contributor_count: u64,
```

Comment:
nit: since we call them participants everywhere else in the pallet
```suggestion
	pub participant_count: u64,
```

Reply:
https://github.com/futureversecom/trn-seed/pull/818/commits/82d50b492b74a0fb360fdcee51ea462a7fc9346c